### PR TITLE
Migrate payout

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: app/migration/settings/billing/[id]/layout.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { BillingHeader } from "./components/billing-header/billing-header";
 function BillingLayout({ children }: PropsWithChildren) {
   const { id } = useParams<{ id: string }>();
   return (
-    <div className="flex flex-col gap-6">
+    <div className="flex flex-col gap-6" id="billing-layout">
```
Comment (haydencleary) [inline app/migration/settings/billing/[id]/layout.tsx:17]: What's the id for ?

File: app/migration/settings/billing/[id]/payment-methods/features/form/form.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  const { data } = BillingProfilesApi.queries.useGetPayoutInfo({
+    params: {
+      id,
+    },
+  });
+
+  return (
+    <Flex
+      direction="col"
+      className="gap-4 divide-y divide-solid divide-greyscale-50/8 [&>div:first-child]:pt-0 [&>div]:pt-4"
```
Comment (haydencleary) [inline app/migration/settings/billing/[id]/payment-methods/features/form/form.tsx:28]: Don't think you need `divide-solid`

File: app/migration/settings/billing/[id]/payment-methods/page.tsx [hunk 3]
```diff
@@ -1,7 +1,147 @@
...
+    },
+  });
+
+  const onSubmit = (formData: TPayoutForm.Data) => {
+    updateUserPayoutInformation(formatToSchema(formData));
+  };
+
+  return (
+    <FormProvider {...formMethods}>
+      <form id="payout-form" onSubmit={handleSubmit(onSubmit)}>
```
Comment (haydencleary) [inline app/migration/settings/billing/[id]/payment-methods/page.tsx:138]: Is the id required ?

File: app/migration/settings/components/form-footer/form-footer.tsx [hunk 4]
```diff
@@ -40,9 +40,12 @@ export function FormFooter({ isPending, hasPreviewButton }: TFormFooter.Props) {
...
       alignItems="center"
       justifyContent="center"
-      className="absolute bottom-0 left-0 right-0 z-50 border-t border-greyscale-50/8 bg-spaceBlue-900 px-4 py-5 shadow-medium xl:px-8"
+      className={cn("border-t border-greyscale-50/8", {
+        "absolute bottom-0 left-0 right-0 z-50 bg-spaceBlue-900 px-4 shadow-medium xl:px-8": isAbsolute,
+        "mt-5 pt-5": !isAbsolute,
+      })}
     >
-      <div className="w-full max-w-7xl px-0 xl:px-8">
+      <div className={cn("w-full", { "w-full max-w-7xl px-0 xl:px-8": isAbsolute })}>
```
Comment (haydencleary) [inline app/migration/settings/components/form-footer/form-footer.tsx:48]: Repeated `w-full`

File: translations/v2/en/pages/settings-billing/settings-billing-payout.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+  "wallets": {
+    "description": "Relevant for rewards in",
+    "ethereum": {
+      "label": "Ethereum wallet address or ENS name",
+      "placeholder": "Ethereum wallet adress"
+    },
+    "starknet": {
+      "label": "Starknet wallet address",
+      "placeholder": "Starknet wallet adress"
```
Comment (haydencleary) [inline translations/v2/en/pages/settings-billing/settings-billing-payout.json:10]:
```suggestion
      "placeholder": "Starknet wallet address"
```

File: translations/v2/en/pages/settings-billing/settings-billing-payout.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
+{
+  "wallets": {
+    "description": "Relevant for rewards in",
+    "ethereum": {
+      "label": "Ethereum wallet address or ENS name",
+      "placeholder": "Ethereum wallet adress"
```
Comment (haydencleary) [inline translations/v2/en/pages/settings-billing/settings-billing-payout.json:6]:
```suggestion
      "placeholder": "Ethereum wallet address"
```

File: translations/v2/en/pages/settings-billing/settings-billing-payout.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    "starknet": {
+      "label": "Starknet wallet address",
+      "placeholder": "Starknet wallet adress"
+    },
+    "optimism": {
+      "label": "Optimism wallet address",
+      "placeholder": "Optimism wallet adress"
```
Comment (haydencleary) [inline translations/v2/en/pages/settings-billing/settings-billing-payout.json:14]:
```suggestion
      "placeholder": "Optimism wallet address"
```

File: translations/v2/en/pages/settings-billing/settings-billing-payout.json [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    "aptos": {
+      "label": "Aptos wallet address",
+      "placeholder": "Aptos wallet adress"
```
Comment (haydencleary) [inline translations/v2/en/pages/settings-billing/settings-billing-payout.json:18]:
```suggestion
      "placeholder": "Aptos wallet address"
```

File: app/migration/settings/billing/[id]/layout.tsx (hunk 1 above)
Comment (alexbeno) [reply app/migration/settings/billing/[id]/layout.tsx:17]: forgot to remove from a test

File: app/migration/settings/billing/[id]/payment-methods/page.tsx (hunk 3 above)
Comment (alexbeno) [reply app/migration/settings/billing/[id]/payment-methods/page.tsx:138]: already here but after a quick look i don't think, let's remove it

**Review by haydencleary — APPROVED** [review]

**Pull request merged**